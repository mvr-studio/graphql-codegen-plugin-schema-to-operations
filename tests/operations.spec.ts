import fs from 'fs'
import path from 'path'

test('matches the operations.graphql snapshot', () => {
  const operationsGql = fs.readFileSync(path.resolve(__dirname, 'operations.graphql'), 'utf8')
  expect(operationsGql).toMatchSnapshot()
})
