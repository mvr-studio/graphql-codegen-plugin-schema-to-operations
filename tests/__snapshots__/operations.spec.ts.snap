// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the operations.graphql snapshot 1`] = `
"query me_query {
  me {
    createdAt
    email
    id
    name
    role
    updatedAt
  }
}

mutation logInUser_mutation($email: String!, $password: String!) {
  logInUser(email: $email, password: $password) {
    jwt
  }
}
"
`;
